Update setup (#31)
LibraryCarpentry/lc-marcedit: setup.md

title: Setup
---

## Getting ready

You need to install MarcEdit and download a data file to follow this lesson.

### Installing and running MarcEdit

MarcEdit is a free, open-source .NET application. You can download MarcEdit from 
[https://marcedit.reeset.net/downloads](https://marcedit.reeset.net/downloads). 

> This lesson has been tested with the version or MarcEdit 7 for Windows and MarcEdit MacOS 3 up to the latest version of 7.5.21.153 for MarcEdit 7 for Windows.

Reviewer: ```suggestion
This lesson has been tested with the following versions: MarcEdit 7 (specifically 7.5.21.153) for Windows and MarcEdit 3 for MacOS.
```

> Packages are available on <https://marcedit.reeset.net/downloads> for Windows, MacOS, and Linux. System requirements for the current release are available on <https://marcedit.reeset.net/downloads>.

Reviewer: ```suggestion
Packages, along with associated system requirements, are available on <https://marcedit.reeset.net/downloads> for Windows and MacOS.
```

> Please download the latest stable version, choosing the "Windows Build Options" or MacOS.

Reviewer: ```suggestion
Please download the latest stable version for your operating system. For Windows, select "Windows Build Options" >> "Download MarcEdit 7.5 (All Users)". For MacOS, select "MacOS".
```

> If choosing the "Windows Build Options", please read through the pop up instructions that explain the differences between a self contained and consolidated installation. The majority of people will choose the self contained installation or a download for one user on a computer in contrast to downloading MarcEdit on a computer accessed by many users. To download the user only installation, choose "Download MarcEdit 7.5 (All Users)".

Reviewer: ```suggestion
The Windows User Only Installation should work for the majority of participants. If you will be using a communal computer, and expect MarcEdit to be used by a number of users on that machine, please read through the "Windows Build Options" pop-up to decide the best installation option for you.
```

> Installation video demostrations are available for both types of installations: self contained <https://youtu.be/FX0r7jIxeN4>, and consolidated <https://youtu.be/7wYo7VoOwMI>. 

Reviewer: ```suggestion
```

Instructions for system administrators has been provided. Also information on managing MarcEdit installations are available <https://marcedit.reeset.net/managing-marcedit-installations>.

If you are using an older version of MarcEdit, it is recommended you upgrade to the latest tested version. 

Notes:

> * The cosolidating build is the only one that currently supports 32-bit systems.

Reviewer: ```suggestion
* The Consolidated Installation for MarcEdit 7.5 for Windows is the only installation option that currently supports 32-bit systems.
```

> * Currently Linux is not supported for version 7.5x only 7.3x for Windows and MacOS 3.2x. 

Reviewer: ```suggestion
```

> * If downloading for Linux, the current Mono is required for installation.

Reviewer: ```suggestion
```

> * When downloading MarcEdit for the first time, the setup wizard will appear. Remember to go through the wizard to set up the applicaiton configrations.

Reviewer: Do they require more guidance for setting up configurations? Should this be moved to the top of the list?

* If the installation is being done via distributed software or via a distribution client like Microsoft's SCCM, please refer to the packaging tool available on GitHub <https://github.com/reeset/marcedit_package_builder>.

System requirements for MarcEdit 7/MarcEdit MacOS 3:
* Operating system: Vista (32 & 64 bit), MacOS 10.10+
* RAM: 8MB minimum; 32 MB+ is recommended
* Disc space for Windows: 34 MB (setup); 15 MB (application)
* Disc space for MacOS: 60 MB (setup); 150 MB (installed)
* For Windows, Windows .NET 4.7.2+ is required.
* For MacOS, Installer, where all dependencies are packaged and installed, is required.

### Downloading the data

You can download [marc_sample_data.mrc](https://github.com/LibraryCarpentry/lc-marcedit/tree/gh-pages/data/marc_sample_data.mrc), which is a MARC binary file that will open in a new browser tab. Be sure to right click or control click in order to save the file (NOTE: In Safari, right click and select **download linked file**; in Chrome and Firefox, right click and select **save link as...**). Make a note of the location (i.e. the folder, your desktop) to which you save the file.

### Getting help

> 

Reviewer: ```suggestion
Installation demonstration videos are available for downloading MarcEdit 7.5 [User Only Installation (Self Contained)](https://youtu.be/FX0r7jIxeN4), and [Consolidated Installations](https://youtu.be/7wYo7VoOwMI) for Windows. 
```
